amp/WEB-INF/src/org/digijava/module/aim/form/TeamAuditForm.java

```diff
@@ -21,7 +22,36 @@
     private int offset;
     private String useraction;
     private String frecuency;
+    private boolean withLogin;
+    private Collection<User> userList;
```

Comment:
this fields are already present in audit logger form, so we should abstract them to FilterAuditLoggerForm